lib/publiccloud/provider.pm

```diff
@@ -19,7 +19,7 @@ use Data::Dumper;
 use Mojo::JSON 'decode_json';
 
 use constant TERRAFORM_DIR     => '/root/terraform';
-use constant TERRAFORM_TIMEOUT => 17 * 60;
+use constant TERRAFORM_TIMEOUT => bmwqemu::scale_timeout((get_var('PUBLIC_CLOUD_SLES4SAP') ? 60 : 17) * 60);
```

Comment:
as for me you somehow duplicating functionality of scale_timeout here - 
```suggestion
use constant TERRAFORM_TIMEOUT => bmwqemu::scale_timeout(60 * 60);
```

or 

```suggestion
use constant TERRAFORM_TIMEOUT => bmwqemu::scale_timeout(17 * 60);
```

and just use SCALE_TIMEOUT variable in appropriate test suite.  But this of course will remove some visibility from code perspective so up to you follow my suggestion or not ...

Reply:
hmm yes I could even let the old code and "just" use `SCALE_TIMEOUT` in my SAP test suites. I will give it a try.

Reply:
Done and it works.